webpack.config.js

```diff
@@ -99,7 +110,11 @@ module.exports = (env, argv) => {
             }
         },
         plugins: [
-            new WebpackDashDynamicImport()
+            new WebpackDashDynamicImport(),
+            new webpack.SourceMapDevToolPlugin({
+                filename: '[name].js.map',
+                exclude: ['async~plotlyjs']
```

Comment:
source mapping plotlyjs accounted for the majority of the build time (15s to build + 30s to map non-plotlyjs chunks + 30s to parse plotlyjs + 3.5mn to map plotlyjs)